server/sonar-server/src/main/java/org/sonar/server/measure/index/ProjectMeasuresIndex.java

```diff
@@ -104,7 +104,8 @@
     .put(FILTER_LANGUAGES,
       (esSearch, facetBuilder, query) -> esSearch.addAggregation(
         facetBuilder.buildStickyFacet(FIELD_LANGUAGES, FILTER_LANGUAGES, query.getLanguages().isPresent() ? query.getLanguages().get().toArray() : new Object[] {})))
-    .put(FILTER_TAGS, (esSearch, facetBuilder, query) -> esSearch.addAggregation(createStickyFacet(FIELD_TAGS, facetBuilder, createTagsFacet())))
+    .put(FILTER_TAGS, (esSearch, facetBuilder, query) -> esSearch.addAggregation(
```

Comment:
I'd extract a method to make this factories more readable. Either the same method for languages and tags or 2 different methods, up to you